Example/Podfile

```diff
@@ -1,13 +1,21 @@
 source 'https://github.com/CocoaPods/Specs.git'
 use_frameworks!
 
-target 'Caishen_Example', :exclusive => true do
+target 'Caishen_Example' do
   pod 'Caishen', :path => '../'
   pod 'CardIO'
 end
 
-target 'Caishen_Tests', :exclusive => true do
+target 'Caishen_Tests' do
   pod 'Caishen', :path => '../'
 
   
 end
+
+post_install do |installer|
```

Comment:
As far as I know, Cocoapods 1.0.x does need it, since it doesn't use the `.swift-version` file that's introduced with version 1.1.x (currently pre-release).